mini-graph-card.js

```diff
@@ -151,6 +151,7 @@ class MiniGraphCard extends LitElement {
       ${this._style()}
       <ha-card
         class='flex'
+        ?bg=${config.background} ?group=${config.group}
```

Comment:
`bg` attribute is not necessary

Reply:
that's what i get for being a sloppy copy/paster